analytics/management/commands/create_ga_report.py

```diff
@@ -0,0 +1,325 @@
+                            help='Other data dimensions to include when querying for specified metrics, such as page path')
+
+        parser.add_argument('--other-filters',
+                            default='ga:source=~.*facebook.com|twitter.com|(direct)|^google$',
+                            dest='otherfilters',
+                            help='Filters for the other data dimensions')
+
+    def handle(self, *args, **options):
+        # Build the service to communicate with the Google Analytics API
+        scopes = ['https://www.googleapis.com/auth/analytics.readonly']
+        key_file_location = utils.get_key_file_location()
+        service_account_email = utils.get_service_account_email()
+        service = utils.get_service(
+            'analytics',
+            'v3',
+            scopes,
+            key_file_location,
+            service_account_email
+        )
+
+        # Get the profile to use with the service
+        profile_id = utils.get_first_profile_id(service)
+
+        # Get parameters
+        start_date = options['startdate']
+        end_date = options['enddate']
+        dimensions = options['dimensions']
+        metrics = options['metrics']
+        other_dimensions = options['otherdimensions']
+        other_filters = options['otherfilters']
+
+        # Get data about the pages from database
+        db_statistics_by_page_url = self._get_statistics_from_database(start_date, end_date)
+
+        # Get data with statistics from Google Analytics
+        ga_statistics, other_ga_stats = self._get_statistics_from_google_analytics(service, profile_id,
+                                                                                   start_date, end_date,
+                                                                                   dimensions, metrics,
+                                                                                   other_dimensions, other_filters)
+
+        # Merge the statistics from the different data sources and print them
+        self._print_statistics(db_statistics_by_page_url, ga_statistics, other_ga_stats)
+
+    def valid_date(self, date_string):
+        try:
+            datetime.strptime(date_string, "%Y-%m-%d")
+        except ValueError:
+            message = "Not a valid date: '{0}'.".format(date_string)
+            raise argparse.ArgumentTypeError(message)
+        else:
+            return date_string
+
+    def _print_statistics(self, db_statistics_by_page_url, ga_statistics_by_page_url, other_ga_stats_by_page_url):
+        header_row = ['Title', 'URL', 'Published Date', 'Days Published', 'Parent Page', 'Category', 'Type',
+                      'Word Count', 'Pageviews (PV)', 'Unique Pageviews (UPV)', 'Avg. Time on Page (in seconds)',
+                      'facebook PV', 'facebook UPV', 'twitter PV', 'twitter UPV', 'direct PV', 'direct UPV',
+                      'google PV', 'google UPV']
+        print '\t'.join([x for x in header_row])
```

Comment:
You might prefer to use stdout over printing explicitly. Check the commands documentation.